AEPServices/Sources/SystemInfoService.swift

```diff
@@ -89,6 +89,8 @@ public protocol SystemInfoService {
 
     /// Returns the application version.
     /// - Return: `String` Application version
+    // TODO: - Include planned deprecation version in message
+    @available(*, deprecated, message: "Please use getApplicationVersionNumber instead")
```

Comment:
Can you also include rename: "getApplicationVersionNumber"?  XCode gives a hint to replace it with new function